src/components/AssetLogo.tsx

```diff
@@ -16,54 +16,161 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
-import { Asset } from '@alephium/sdk'
 import { ALPH } from '@alephium/token-list'
-import { Ghost } from 'lucide-react'
-import styled, { css } from 'styled-components'
+import { HelpCircle } from 'lucide-react'
+import { useEffect } from 'react'
+import ReactTooltip from 'react-tooltip'
+import styled, { css, useTheme } from 'styled-components'
 
+import { useAssetMetadata } from '@/api/assets/assetsHooks'
 import AlephiumLogoSVG from '@/images/alephium_logo_monochrome.svg'
 
 interface AssetLogoProps {
-  asset: Pick<Asset, 'id' | 'logoURI' | 'name'>
+  assetId: string
   size: number
   showTooltip?: boolean
   className?: string
 }
 
-const AssetLogo = ({ asset, size, showTooltip, className }: AssetLogoProps) => (
-  <div className={className} data-tip={showTooltip && (asset.name || asset.id)}>
-    {asset.logoURI ? (
-      <LogoImage src={asset.logoURI} />
-    ) : asset.id === ALPH.id ? (
-      <LogoImage src={AlephiumLogoSVG} />
-    ) : (
-      <Ghost size={size * 0.6} />
-    )}
-  </div>
+const AssetLogo = (props: AssetLogoProps) => {
+  const { assetId, showTooltip, className } = props
+
+  const theme = useTheme()
+
+  const metadata = useAssetMetadata(assetId)
+
+  const assetType = metadata.type
+
+  useEffect(() => {
+    metadata && ReactTooltip.rebuild()
+  }, [metadata])
```

Comment:
I am surprised that this is still needed! Why is it that we don't need it on the DW anymore? Are we using an updated version?

Reply:
Yep. Didn't have the courage to update to v5 just yet.